documentstore_migracao/main/tools.py

```diff
@@ -34,21 +34,26 @@ def tools_parser(sargs):
     )
 
     # CONSTRUCTOR XML
+    parents_construction = paths_parser(
+        sargs,
+        **{
+            "d_source": config.get("VALID_XML_PATH"),
+            "h_source": "Pasta onde esta os xml para transformação, default: VALID_XML_PATH",
+            "d_desc": config.get("CONSTRUCTOR_PATH"),
+            "h_desc": "Pasta onde sera salvo os XML alterados, default: CONSTRUCTOR_PATH",
+        }
+    )
+    parents_construction.add_argument(
+        "--in-place", type=bool, default=False,
```

Comment:
Aqui poderia ser `action='store_false'`? Assim poderíamos informar no comando somente `--in-place`.

Reply:
Teria que ser um `store_true` então teríamos como default o valor `False`

Reply:
@patymori @joffilyfe As correções foram feitas